src/three/TilesRenderer.js

```diff
@@ -133,6 +134,16 @@ export class TilesRenderer extends TilesRendererBase {
 
 		} else {
 
+			const boundingSphere = cached.sphere;
+
+			if ( boundingSphere ) {
```

Comment:
If there is no cached bounding box, return a bounding box based on the cached bounding sphere

Reply:
Great thanks! We should do the same for the `getOrientedBounds` function.

Reply:
I was wondering if you could help me with the orientedBounds logic.
I'm not sure how to calculate the obb matrix based on the boundingSphere. Any ideas or recommendations?

Reply:
Yup! Computing an oriented bounding box I think is more work than is needed. I'm thinking in this case it's enough to just produce the same AABB and identity matrix. At least then the use won't get an empty result when they need a valid bounds to work with. In the future I think these functions need to be reconsidered so users can make sure they're using the natively defined bounds if necessary, I think.